apps/playground-sveltekit/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "sveltekit-nextauth",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "svelte-kit dev",
+    "build": "svelte-kit build",
+    "package": "svelte-kit package",
```

Comment:
Without knowing the details yet, I am fairly confident we will have our own way of packaging, or at least not as part of the playground.

If possible, could we remove anything related to packaging from this directory?

Reply:
Yeah I think we can remove this since we'll have our own packaging